src/rdb.c

```diff
@@ -1514,10 +1521,8 @@ robj *rdbLoadObject(int rdbtype, rio *rdb, sds key) {
             double score;
             zskiplistNode *znode;
 
-            if ((sdsele = rdbGenericLoadStringObject(rdb,RDB_LOAD_SDS,NULL)) == NULL) {
-                decrRefCount(o);
```

Comment:
This shouldn't be removed. Missing adjustPmemThresholdCycle():
```
            if (zsetlen%100 == 0) {
                adjustPmemThresholdCycle();
            }

            if ((sdsele = rdbGenericLoadStringObject(rdb,RDB_LOAD_SDS,NULL)) == NULL) {
                decrRefCount(o);
                return NULL;
            }
```

Reply:
done